scripts/generate_schemas.py

```diff
@@ -227,6 +228,24 @@ def write(self):
                 c = classes[name]
                 c.write(f)
 
+            # can't just use pprint library because
```

Comment:
Depending on the lengths of the names of the classes, pprint may choose to indent in a way that is not pep8-compliant.  But just more generally, pprint doesn't seem to want to respect pep8 -- pprint seems to be a library meant for human readability, not compliance after being piped to a generated code file.